pyfolio/plotting.py

```diff
@@ -1038,6 +1038,39 @@ def plot_turnover(returns, transactions, positions,
     return ax
 
 
+def plot_daily_turnover_hist(transactions, positions,
+                             ax=None, **kwargs):
+    """Plots a histogram of daily turnover rates.
+
+    Parameters
+    ----------
+    transactions : pd.DataFrame
+        Daily transaction volume and dollar ammount.
+         - See full explanation in tears.create_full_tear_sheet.
+    positions : pd.DataFrame
+        Daily net position values.
+         - See full explanation in tears.create_full_tear_sheet.
+    ax : matplotlib.Axes, optional
+        Axes upon which to plot.
+    **kwargs, optional
+        Passed to seaborn plotting function.
+
+    Returns
+    -------
+    ax : matplotlib.Axes
+        The axes that were plotted on.
+
+    """
+
+    if ax is None:
+        ax = plt.gca()
+    turnover = pos.get_turnover(transactions, positions, period=None)
+    sns.distplot(turnover, ax=ax, **kwargs)
+    ax.set_title('Distribution of Daily Turnover Rates')
```

Comment:
Minor: I want to start refactoring this pattern to: `ax.set(title='...', xlabel='...')`.